strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/maven/MavenArtifactController.java

```diff
@@ -236,11 +234,11 @@ private boolean probeForDirectoryListing(Repository repository,
                                              String path)
     {
         String filePath = path.replaceAll("/", Matcher.quoteReplacement(File.separator));
-
+        
```

Comment:
Could you please revert this file to the previous state ? All your changes are related with spaces corrections, so they just bring a noise.